test-case/check-pause-resume.sh

```diff
@@ -91,14 +91,14 @@ set i 1
 expect {
     "*#*+*\%" {
         set sleep_t [expr int([expr rand() * $rnd_range]) + $rnd_min ]
-        puts "\r(\$i/$repeat_count) Wait for \$sleep_t ms before pause"
+        puts "\r(\$i/$repeat_count) Wait for \$sleep_t ms after pause"
```

Comment:
@stripes416 I think you may confuse with the logic here.
At the moment the log output, we are playing and wait the the time. Then we inject space to enter pause.
So the log here is right to show before pause.

Reply:
@xiulipan are you sure? because the `send " "` is what actually pauses the playback and `after `$sleep_t` is what causes the sleep period. We're pausing before we sleep.